finance/sc-im/Portfile

```diff
@@ -0,0 +1,27 @@
+PortSystem          1.0
+name                sc-im
+version             0.8.2
+license             BSD
+maintainers         nkorth
```

Comment:
this is interpreted as an e-mail address, which is probably not what you intended. If you want to specify your GitHub handle you should include the `@`. Also, is possible please add `openmaintainer` which would allow others to make minor updates/fixes to the Portfile without your explicit permission.